src/code/z_viszbuf.c

```diff
@@ -0,0 +1,98 @@
+/**
+ * @file z_viszbuf.c
+ * Description: Visualise the z-buffer (AKA depth buffer), using cycling RGBA or a single fading color.
+ *
+ * useRgba:
+ * - false will produce a monotonic fade from primColor to envColor as depth increases
+ * - true produces vibrant almost-periodic-looking bands.
+ *
+ * In both cases this occurs because of the format the depth information takes: it is 18-bit, and is a positive
+ * floating-point number with
+ *     bbb      mmmmmmmmmmm dd|dd
+ *     exponent mantissa    dz value (only first 16 bits visible to CPU)
+ *
+ * IA:
```

Comment:
```suggestion
 * When reading the z-buffer as IA16:
```
or something similar

IA*16* matters of course
and I think it's worth making it clearer the z-buffer is being read/loaded/interpreted? as an image